data/ocua_17-18/week4_game2.json

```diff
@@ -314,7 +314,7 @@
           "type": "PASS"
```

Comment:
Should change this Jim reception to Pindur also.

Reply:
yes it should! nice catch